nevergrad/parametrization/data.py

```diff
@@ -332,19 +341,39 @@ class Scalar(Array):
 
     Parameters
     ----------
-    init: float
+    init: optional float
         initial value of the scalar
+    lower: optional float
+        minimum value if any
+    upper: optional float
+        maximum value if any
     mutable_sigma: bool
         whether the mutation standard deviation must mutate as well (for mutation based algorithms)
 
-    Note
-    ----
-    More specific behaviors can be obtained throught the following methods:
-    set_bounds, set_mutation
+    Notes
+    -----
+    - if both lower and upper bounds are provided, sigma will be adapted so that the range spans 6 sigma.
+      Also, if init is not provided, it will be set to the middle value.
+    - More specific behaviors can be obtained throught the following methods:
+     set_bounds, set_mutation
     """
 
-    def __init__(self, init: float = 0.0, mutable_sigma: bool = True) -> None:
+    def __init__(
+        self,
+        init: float = 0.0,
+        lower: tp.Optional[float] = None,
+        upper: tp.Optional[float] = None,
+        mutable_sigma: bool = True
+    ) -> None:
+        bounded = all(a is not None for a in (lower, upper))
+        if bounded:
+            if init is None:
+                init = (lower + upper) / 2.0
         super().__init__(init=np.array([init]), mutable_sigma=mutable_sigma)
+        if bounded:
+            self.set_mutation(sigma=(upper - lower) / 6)  # type: ignore
```

Comment:
@teytaud 
what do you think of these defaults for scalars:
if bound lower and upper bound are provided:
- sigma is set to a sixth of b - a (so that the full range is covered from start)
- we also set `full_range_sampling=True` which means that parametrization.sample() will sample uniformly within the bounds ( this is used by some new algorithms)
- if init is not provided, it is set it is set to the middle point

Reply:
If neither or only one is provided, we are back to normal and the user need to use the Scalar methods to update sigma

Reply:
Also forgot to mention, by default it's using clipping, but once again to use a different method one can set it manually through the set_bounds method